### ISSUE TITLE
Unable to add new category

### ISSUE THREAD

**Issue by jcs0596223** [jobisoft/CategoryManager#69]
I am new to Thunderbird and have just tried out the Category Manager. Three categories are listed but I am unable to add any new ones. I can't actually remember where the categories were originally set.

Attempting to add a new category displays a dialog box with the label - see screenshot
![Screenshot 2021-06-07 125441](https://user-images.githubusercontent.com/85493566/121012562-bff9c500-c78f-11eb-8da2-1db14f75e88c.png)

I can type a new category but it does not create  new one.

I am using  exchange sync to connect to my Outlook.com account and this is successfully pulling in my address book.

I have tbsync and Exchange activesink installed.

In case it is relevant I have had trouble installing previous add-ons as McAfee appears to be blocking the installation. I had to download the manual install file and install that. In one case (I can't remember which I had to use a VM without McAfee and install via that route to get the manual install file as a direct download was not available.

A new computer is about to be delivered where the category manager will be highly desirable

**Comment by jobisoft** [comment]
Uups. Looks like a regression introduced by the latest update.

**Comment by jcs0596223** [comment]
Is the previous version still available? Or will the problem be soon fixed?

 I was hoping to finish setting up the new computer tomorrow

**Comment by jcs0596223** [comment]
I have now found the previous version and after having a serious argument with McAfee AV that insists it is a virus and quarantines the installer have managed to install it - and it works and does what I need.

For your information McAfee is quarantining **Attachment!script.a** which I am guessing is part of the install process. Your later version did not raise this error.

My workaround is to use a W10 Virtual Machine (which does not have McAfee AV installed and to use that to install into Thunderbird. I then find the categorymanager-4.1-tb.xpi file, copy from the VM to my real computer and use the install from file option. I have had to use the same trick already to install other add-ins. Messy but it seems to work

**Comment by jobisoft** [comment]
Reopen, as the bug is still there.

**Comment by PeetThesing** [comment]
Does anyone knows a workaround for this bug other than the virtual machine? I have to install this add-on on several computers for our nextcloud-sychronosation.

**Comment by jcs0596223** [comment]
I may have confused you with my explanation. I was unable to install the previous version due to my McAfee antivirus software blocking installation of ALL Thunderbird add-ins. The virtual machine was my method of getting the installation to work to bypass McAfee. I have now removed McAfee from my computer

I have now removed McAfee and can confirm that the previous version does work and should install OK (unless you are using McAfee antivirus of course)

One caution is that if you allow automatic updates it automatically replaces the working older version by the newer flawed version. I have switched automatic updates off but have not checked whether there is a new version available recently to replace the bug I opened this thread with.

The add-in does exactly what I wanted it to do!
